data/templates/photon-os-ks

```diff
@@ -0,0 +1,35 @@
+{
+    "hostname": "test_hostname",
```

Comment:
Use template hostname: <%=hostname%> and the `hostname` may be optional, see https://github.com/RackHD/on-http/blob/master/data/templates/centos-ks#L127